server/models/address.go

```diff
@@ -27,5 +27,5 @@ type Address struct {
 }
 
 type AddressesList struct {
-	Adresses []*Address `json:"adresses"`
+	Addresses []*Address `json:"addresses"`
```

Comment:
I checked to see if this was in the public API, but there aren't actually any usages anywhere. Can this struct be removed?